src/cargo/util/toml/mod.rs

```diff
@@ -768,6 +822,8 @@ fn normalize_dependencies<'a>(
                     }
                 }
             }
+            normalize_path_dependency(gctx, d, workspace_root, features)
+                .with_context(|| format!("resolving path dependency ({name_in_toml})"))?;
```

Comment:
```suggestion
                .with_context(|| format!("resolving path dependency `{name_in_toml}`"))?;
```
seems weird to use a parenthetical when its not needed

Reply:
ok